lib/pure/parsecsv.nim

```diff
@@ -15,8 +15,11 @@
 ##
 ## .. code-block:: nim
 ##   import os, parsecsv, streams
```

Comment:
Since the example showcases how to use procedures from this module, my preference is to show exactly what was imported from other modules so there is less confusion about it. I would make imports like this:

```nim
import parsecsv
from os import paramStr
from streams import newFileStream
```

Reply:
Thx. I agree yours.
I'll change imports.